config/config.go

```diff
@@ -116,5 +121,26 @@ func Load(path string, netMode NetMode) (Config, error) {
 		return Config{}, errors.Wrap(err, "Problem unmarshaling config json data")
 	}
 
+	setConfigDefault(&config)
+
 	return config, nil
 }
+
+func setConfigDefault(cfg *Config) {
+	ConfigDefault = cfg
+}
+
+func (s SystemFee) TryGetValue(txType transaction.TXType) util.Fixed8 {
+	switch txType {
+	case transaction.EnrollmentType:
+		return util.NewFixed8(s.EnrollmentTransaction)
+	case transaction.IssueType:
+		return util.NewFixed8(s.IssueTransaction)
+	case transaction.PublishType:
+		return util.NewFixed8(s.PublishTransaction)
+	case transaction.RegisterType:
+		return util.NewFixed8(s.RegisterTransaction)
+	default:
+		return util.NewFixed8(0)
```

Comment:
we must fix `NewFixed8` method:

```go
// NewFixed8 return a new Fixed8 type multiplied by decimals.
func NewFixed8(val int64) Fixed8 {
	return Fixed8(decimals * val)
}
```

Reply:
or mb to:
```go
// NewFixed8 return a new Fixed8 type multiplied by decimals.
func NewFixed8(val float64) Fixed8 {
	return Fixed8(int64(decimals * val))
}
```

Reply:
why?

Reply:
see  [there](https://github.com/CityOfZion/neo-go/pull/135#discussion_r255551261)